projects/rero/ng-core/src/lib/record/editor/editor.component.ts

```diff
@@ -40,6 +40,9 @@ import { isEmpty, orderedJsonSchema, removeEmptyValues } from './utils';
 import { LoadTemplateFormComponent } from './widgets/load-template-form/load-template-form.component';
 import { SaveTemplateFormComponent } from './widgets/save-template-form/save-template-form.component';
 
+export interface JSONSchema7 extends JSONSchema7Base {
+  form: any;
+}
```

Comment:
maybe a space after the interface, as this is not linked to component. Or better in a specific file.